ComfortZoneApp/app/src/main/java/com/example/comfortzone/fragments/FlightFragment.java

```diff
@@ -162,7 +173,9 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
 
             @Override
             public void afterTextChanged(Editable s) {
-                flightsAdapter.searchCity(s.toString().toLowerCase(Locale.ROOT), db.weatherDao().getAll());
+                String city = Normalizer.normalize(s, Normalizer.Form.NFD);
+                city = city.replaceAll("[^\\p{ASCII}]", "");
```

Comment:
nice! if our cities also have accented characters we should do it for them too just for future reference